doc/tutorial/09_retention.md

```diff
@@ -0,0 +1,46 @@
+# Retention Policy
+
+This tutorial will show you how to configure retention to retain backups.
+
+## Preface
+
+This tutorial assumes that you have an installation of PostgreSQL 10+ and pgmoneta.
+
+See [Install pgmoneta](https://github.com/pgmoneta/pgmoneta/blob/main/doc/tutorial/01_install.md)
+for more detail.
+
+## Retention Setup
+
+In `pgmoneta.conf`, you can use `retention = 7, 4, 12, 5` to configure pgmoneta to retain backups
+within the nearest 7 days, 4 weeks, 12 months and 5 years. Specifically, pgmoneta will retain
+all the backups within the nearest 7 days, the latest backup on each Monday within the nearest 4 weeks,
+the latest backup on the first day of each month in the last 12 months and the latest backup on the first
+day of each year in the last 5 years. If you input more than 4 values, pgmoneta will only read the first 4.
+
+
+There are a lot of ways to leave a parameter unspecified. You can omit trailing parameters, 
+for example, you can configure retention as `retention = 7, 4`,
+then pgmoneta will only keep all the backups within the nearest 7 days and the latest backup on each Monday
+within the nearest 4 weeks. The retention months and years will not be subject to configuration validation.
+You can also leave some parameter in between unspecified by setting it to special characters / values, 
+currently we allow: `x`, `X`, `-`, whitespaces(spaces or tabs) or any non-positive value. 
+However, you should not leave it empty.
+
+Here's a list of some valid input examples:
+1. Setting it to non-positive values: `retention = 7, -1, 12`, `retention = 7, -2, 12`, `retention = 7, 0, 12` ...
```

Comment:
I don't think negative is a good idea.

User input should be 0+, -, x or X

Reply:
And maybe empty

Reply:
Empty is tricky as we talked about, I currently don't allow that, I allow whitespaces though. For negative values I agree it's a little weired. Do you want to keep -1 though? I'm hesitating because previously you mentioned that you want negative values to mean unspecified: https://github.com/pgmoneta/pgmoneta/pull/117#discussion_r1158914634

Reply:
I think negative is an internal value for unspecified.

For users it should be 0+ , -, x or X